mp/src/public/vgui_controls/ProgressBar.h

```diff
@@ -20,87 +20,108 @@ namespace vgui
 class ProgressBar : public Panel
 {
-	DECLARE_CLASS_SIMPLE( ProgressBar, Panel );
+    DECLARE_CLASS_SIMPLE(ProgressBar, Panel);
 
 public:
-	ProgressBar(Panel *parent, const char *panelName);
-	~ProgressBar();
+    ProgressBar(Panel *parent, const char *panelName);
+    ~ProgressBar();
 
-	// 'progress' is in the range [0.0f, 1.0f]
-	MESSAGE_FUNC_FLOAT( SetProgress, "SetProgress", progress );
-	float GetProgress();
-	virtual void SetSegmentInfo( int gap, int width );
+    // 'progress' is in the range [0.0f, 1.0f]
+    MESSAGE_FUNC_FLOAT(SetProgress, "SetProgress", progress);
+    float GetProgress();
+    virtual void SetSegmentInfo(int gap, int width);
     virtual void SetProgressText();
 
-	// utility function for calculating a time remaining string
-	static bool ConstructTimeRemainingString(OUT_Z_BYTECAP(outputBufferSizeInBytes) wchar_t *output, int outputBufferSizeInBytes, float startTime, float currentTime, float currentProgress, float lastProgressUpdateTime, bool addRemainingSuffix);
+    // utility function for calculating a time remaining string
+    static bool ConstructTimeRemainingString(OUT_Z_BYTECAP(outputBufferSizeInBytes) wchar_t *output, int outputBufferSizeInBytes, float startTime, float currentTime, float currentProgress, float lastProgressUpdateTime, bool addRemainingSuffix);
 
-	void SetBarInset( int pixels );
-	int GetBarInset( void );
-	void SetMargin( int pixels );
-	int GetMargin();
+    void SetBarInset(int pixels);
+    int GetBarInset();
+    void SetMargin(int pixels);
+    int GetMargin();
 
     void SetShouldDrawPercentString(bool bDraw);
-	
-	virtual void ApplySettings(KeyValues *inResourceData);
-	virtual void GetSettings(KeyValues *outResourceData);
-    void InitSettings() OVERRIDE;
 
-	// returns the number of segment blocks drawn
-	int GetDrawnSegmentCount();
+    virtual void ApplySettings(KeyValues *inResourceData);
+    virtual void GetSettings(KeyValues *outResourceData);
+    void InitSettings() override;
 
-	enum ProgressDir_e
-	{
-		PROGRESS_EAST,
-		PROGRESS_WEST,
-		PROGRESS_NORTH,
-		PROGRESS_SOUTH
-	};
+    // returns the number of segment blocks drawn
+    int GetDrawnSegmentCount();
 
-	int GetProgressDirection() const { return m_iProgressDirection; }
-	void SetProgressDirection( int val ) { m_iProgressDirection = val; }
+    enum ProgressDir_e
+    {
+        PROGRESS_EAST,
+        PROGRESS_WEST,
+        PROGRESS_NORTH,
+        PROGRESS_SOUTH
+    };
+
+    int GetProgressDirection() const { return m_iProgressDirection; }
+    void SetProgressDirection(int val) { m_iProgressDirection = val; }
+
+    void SetSubdivMarksVisible(bool bVisible) { m_bSubdivMarksVisible = bVisible; }
+    void SetSubdivMarksColor(Color color) { m_SubdivMarksColor = color; }
+    void SetSubdivMarksCount(int count) { m_iSubdivMarksCount = count; }
+    void SetSubdivMarksWidth(int width) { m_iSubdivMarksWidth = width; }
+    void SetSubdivMarksHidden(bool bHidden) { m_bSubdivMarksHidden = bHidden; }
```

Comment:
As this directly conflicts with the `SetSubdivMarksVisible` member variable / method names, consider renaming this to `SetSubdivMarksVisible` and change the one used inside of the settings to `m_bSubdivMarksEnabled`. "Enable" here means the user *wants* them, and we temporarily *hide* them (visibility), so it should be much clearer for the future.